src/plugins/intel_gna/src/gna_plugin_config.cpp

```diff
@@ -267,6 +267,8 @@ OPENVINO_SUPPRESS_DEPRECATED_START
 OPENVINO_SUPPRESS_DEPRECATED_END
         } else if (key == CONFIG_KEY(LOG_LEVEL) || key == ov::log::level) {
             gnaFlags.log_level = ov::util::from_string(value, ov::log::level);
+        } else if (key == CONFIG_KEY(CACHE_DIR) || key == ov::cache_dir) {
```

Comment:
I think we should use only new ov property there

Reply:
ok, I will remove key == CONFIG_KEY(CACHE_DIR)